src/main/kotlin/ar/edu/unahur/obj2/caralibro/Usuario.kt

```diff
@@ -1,11 +1,35 @@
 package ar.edu.unahur.obj2.caralibro
 
+import java.lang.Exception
+
 class Usuario {
   val publicaciones = mutableListOf<Publicacion>()
+  val amigos = mutableListOf<Usuario>()
+
 
   fun agregarPublicacion(publicacion: Publicacion) {
     publicaciones.add(publicacion)
   }
 
+  fun agregarAmigo(amigo: Usuario) {amigos.add(amigo)}
+
+  fun cantidadAmigos() = amigos.size
+
   fun espacioDePublicaciones() = publicaciones.sumBy { it.espacioQueOcupa() }
+
+  fun esMasAmistoso_Que_(unUsuario: Usuario, otroUsuario: Usuario) {
+    unUsuario.amigos.size > otroUsuario.amigos.size
+  }
```

Comment:
Está mal que este método tenga dos parámetros, porque lo que se quiere es comparar al usuario actual (`this`) contra otro. Sería así:

```suggestion
  fun esMasAmistoso_Que_(otroUsuario: Usuario) {
    amigos.size > otroUsuario.amigos.size
  }
```

Reply:
Este requerimiento no está testeado.